tests/link_metrics_test.py

```diff
@@ -0,0 +1,49 @@
+#
+# Copyright 2016 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import os
+import unittest
+
+from glyphsLib import load_to_ufos
+
+
+def glyphs_file_path():
+    return os.path.join(os.path.dirname(__file__), "data", "LinkMetrics.glyphs")
+
+
+class UtilTest(unittest.TestCase):
```

Comment:
you are using plain `assert .. == ...` like in pytest-style tests but then you're sublcassing from unittest.TestCase.. Either you subclass TestCase and then use unittest-style self.assertEqual(etc.) or you simply use pytest-style testing and remove the subclassing from unittest.TestCase. You can still group pytest tests as instance methods by creating the test class and naming it such that it _starts_ with the string "Test" (see https://docs.pytest.org/en/reorganize-docs/new-docs/user/naming_conventions.html). We can configure pytest so that it also recognizes class names that ends with "Test" but it's simpler if you rename it `TestUtil`.
I prefer that you use pytest-style for newly added tests.
The test runner in pytest anyway and it understand both. But mixing ways the two like you do here is incorrect.